examples/pcolor_example.py

```diff
@@ -18,12 +18,44 @@
 
 x = psp.nddata(r_[-5, -2, -1, -0.5, 0, 0.5, 5], "x")
 y = psp.nddata(3 * r_[-5, -2, -1, -0.5, 0, 0.5, 5], "y")
-z = plt.exp(-((y - 2) ** 2) - (x - 0) ** 2 / 2) + 1j * x
+z1 = plt.exp(-((y - 2) ** 2) - (x - 0) ** 2 / 2) + 1j * x
+z2 = 10 * z1
+# {{{ plot the smaller data
 plt.figure()
-plt.suptitle("colorscales dependent")
-z.dimlabels
-z.pcolor()
+z1.C.pcolor(scale_independently=True)
+# }}}
+# {{{ plot the larger data
 plt.figure()
+z2.pcolor(scale_independently=True)
+
+
+# }}}
```

Comment:
surprisingly, these don't work well.  Notice that I even tried copying the first dataset before running pcolor.

If I comment out either call to pcolor, the other one works correctly.